tests/dummy_data_generator/test_data_generator.py

```diff
@@ -0,0 +1,251 @@
+                'library_name': 'Test_library_name_0',
+                'library_strand': random.choice(library_strand_list),
+                'is_paired_end': random.choice(is_paired_end_list),
+                'platform': random.choice(platform_list),
+                'instrument_model': random.choice(instrument_model_list),
+                'max_insert_size': random.choice([300, 350, 500]),
+                'mean_insert_size': random.randint(300,500),
+                'mean_depth': random.randint(40,60),
+                'total_reads': random.randint(400,1000),
+                'mean_read_length': random.randint(400,1000)
+            }
+            e_list.append(
+                SequencingExperiment(
+                    **e_data,
+                    genomic_files=self._create_genomic_files()))
+        return e_list
+
+    def _create_genomic_files(self, total=4):
+        file_format_list = ['.cram', '.bam', '.vcf']
+        gf_list = []
+        for i in range(total):
+            kwargs = {
+                'file_name': 'file_{}'.format(i),
+                'file_type': 'submitted aligned read',
+                'file_format': random.choice(file_format_list),
+                'file_url': 's3://file_{}'.format(i),
+                'md5sum': str(uuid.uuid4()),
+            }
+            gf_list.append(GenomicFile(**kwargs))
+        return gf_list
+
+    def _create_demographics(self):
+        race_list = [
+            "White",
+            "Black or African American",
+            "Asian",
+            "Native Hawaiian or Other Pacific Islander",
+            "American Indian or Alaska Native",
+            "Other",
+            "Unavailable",
+            "Not Reported",
+            "not allowed to collect"]
+        ethnicity_list = ["hispanic or latino",
+                          "not hispanic or latino",
+                          "Unknown",
+                          "not reported",
+                          "not allowed to collect"]
+        gender_list = ['female',
+                       'male',
+                       'unknown',
+                       'unspecified'
+                       'not reported']
+        data = {
+            'external_id': 'demo_id',
+            'race': random.choice(race_list),
+            'ethnicity': random.choice(ethnicity_list),
+            'gender': random.choice(gender_list)
+        }
+        return Demographic(**data)
+
+    def _create_diagnoses(self, total=4):
+        ref_file = open('dataservice/util/data_gen/diagnoses.txt', 'r')
+        reader = csv.reader(ref_file)
+        diagnosis_list = []
+        for line in reader:
+            diagnosis_list.append(line[0])
+        progression_or_recurrence_list = ['yes', 'no', 'unknown']
+        diag_list = []
+        for i in range(total):
+
+            data = {
+                'external_id': 'diagnosis_{}'.format(i),
+                'diagnosis': random.choice(diagnosis_list),
+                'progression_or_recurrence': 
```

Comment:
Remove progression_or_recurrence. This field is no longer in the Diagnosis model